src/node_util.cc

```diff
@@ -146,6 +148,36 @@ void WatchdogHasPendingSigint(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void CreatePromise(const FunctionCallbackInfo<Value>& args) {
+  Local<Context> context = args.GetIsolate()->GetCurrentContext();
+  auto maybe_resolver = Promise::Resolver::New(context);
+  if (!maybe_resolver.IsEmpty())
+    args.GetReturnValue().Set(maybe_resolver.ToLocalChecked());
+}
```

Comment:
I *think* so. It’s using V8s internal promise creation mechanism instead of the full Promise constructor … I would assume the latter is built upon the former, but I haven’t checked that. If you want, I can try to look it up.